app/src/main/java/com/mathroule/demoapp/MainActivity.java

```
@@ -28,6 +31,31 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            setButonLable(text);
+
+            if (text == "something") {
+                setButonLable("something");
+            } else           if (text == "something-else") {
+                setButonLable("something-else");
+            }else           if (text == "something-else") {
+                setButonLable(null);
+            }
+
+        } catch (Exception e) {
```

<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/dad3675473f7e2225c321f103b6011e5dcce7d96/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L47"></span>Empty catch block.</td>
    </tr>
  </tbody>
</table>